2_process.R

```diff
@@ -61,5 +63,40 @@ p2_targets_list <- list(
         purrr::map(.,summarize_paired_comids) %>%
         bind_rows()
     }
-  )
+  ),
+  
+  # Extract COMID and respective area for LC estimation below
```

Comment:
Added the following 4 targets here for estimating proportions of LC at the PRMS scale.